cluster-autoscaler/cloudprovider/azure/test/e2e/e2e_suite_test.go

```diff
@@ -76,7 +82,12 @@ func TestE2E(t *testing.T) {
 var _ = BeforeSuite(func() {
```

Comment:
hmmm I wonder if we also should have an `AfterSuite`, or if we assume that the resources will be properly deleted by some other method... 
I'll check on this, as well.